Formula/png++.rb

```diff
@@ -12,9 +11,74 @@ class Pngxx < Formula
     sha256 "f2e242ee428f191645418a9897eb2fd729408dd67d04e7af4cefc7dcb5715250" => :mavericks
   end
 
+  head do
+    url "svn://svn.savannah.nongnu.org/pngpp/trunk"
+  end
+
   depends_on "libpng"
 
+  patch :DATA
```

Comment:
It's submitted to the upstream and I have written some explanation in the patch header. I'll move it to here.